src/ProcessManager/RefundPaymentProcessManager.php

```diff
@@ -56,7 +56,8 @@ public function __invoke(UnitsRefunded $event): void
             $event->orderNumber(),
             $event->amount(),
             $event->currencyCode(),
-            $event->paymentMethodId()
+            $event->paymentMethodId(),
+            1
```

Comment:
I suppose this is currently hardcoded like this to support single-payment orders?